### DISCUSSION TITLE
Extract deep object

### DISCUSSION THREAD

**Discussion by damanis** [yamafaktory/jql#254] [Q&A] [answered]
Sorry, maybe I missed something, but how extract, for example, all `local` fields from `ip -j a` output (in Linux)?

**Comment by yamafaktory** [comment]
Hi @damanis ,
I'll have a look at it as soon as I'm in front of my laptop 🙂.

**Comment by yamafaktory** [comment, marked as answer]
Hi @damanis :) 

`ip -j a | jql '|>"addr_info"<|..|>"local"'`

**Reply by damanis** [reply]
Thank you!
